File: triplets.py
def tripleSum(arr , x):
    count = 0
    n = len(arr)
    for i in range(0, n-2):
        for j in range(i+1, n-1):
            for k in range(j+1, n):
                if arr[i] + arr[j] + arr[k] == x:
                    count += 1
    return count

my_list = [1,4,5,6,3]
target = 12
output = tripleSum(my_list , target)
print(output)

File: Triplets.java
public class Triplets {
    static int tripletSum(int[] arr, int x) {
        int count = 0;
        for (int i = 0; i < arr.length; i++) {
            for (int j = i + 1; j < arr.length; j++) {
                for (int k = j + 1; k < arr.length; k++) {
                    if (arr[i] + arr[j] + arr[k] == x) {
                        count = count + 1;
                    }
                }
            }
        }
        return count;
    }

    public static void main(String[] args) {
        int arr[] = { 1, 4, 5, 6, 3 };
        int target = 12;
        int output = tripletSum(arr, target);
        System.out.println(output);
    }
}